tests/sanity/tests/liveness/login.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { test } from '@playwright/test'
```

Comment:
@JasminMus Let's move Liveness checks for another folder or probable event repository. Since we plan to have multiple demo workspaces, and it will be hard to maintain in this repository. Probable we could for now move it into our second repository with a real deployment. Thoughts?

Reply:
@haiodo sure we can move this to another repository, I left it here because I didn't wanna duplicate things such as locators, playwright setup etc, we would just need a new workflow that will do the install, not even use docker and hit the production.
But I am fine with moving it to another repository, I will just need a bit of guidance to where.